packages/flutter_adaptive_scaffold/lib/src/slot_layout.dart

```diff
@@ -20,11 +20,17 @@ class SlotLayout extends StatefulWidget {
   static SlotLayoutConfig? pickWidget(
       BuildContext context, Map<Breakpoint, SlotLayoutConfig?> config) {
     SlotLayoutConfig? chosenWidget;
+    SlotLayoutConfig? chosenFallbackWidget;
     config.forEach((Breakpoint breakpoint, SlotLayoutConfig? pickedWidget) {
       if (breakpoint.isActive(context)) {
-        chosenWidget = pickedWidget;
+        if (breakpoint.platform != null) {
```

Comment:
This fixes a bug where the first breakpoint is picked, instead of the most specific one.